base/echo/base/deployment.yml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: echo
+  labels:
+    app.kubernetes.io/name: echo
+    app.kubernetes.io/instance: echo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: echo
+      app.kubernetes.io/instance: echo
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: echo
+        app.kubernetes.io/instance: echo
+    spec:
+      containers:
+      - env:
+        - name: JAVA_OPTS
+          value: -XX:MaxRAMPercentage=80.0
```

Comment:
Do we think this increase from the Halyard-set default is okay for Echo as well as Clouddriver?

Reply:
Yes, I deliberately increased it as it seemed silly to limit to 50% of the RAM when running in a container with nothing else there, so I'd suggest doing this for all the services.